config-examples/users.yaml

```diff
@@ -0,0 +1,4 @@
+users:
+  - name: default
+    password: default
```

Comment:
never ever store password in plain string format.

Reply:
Of course (: It's the first iteration to figure out the whole feature organization.